internal/resources/azure/iothub.go

```diff
@@ -0,0 +1,104 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+// IoTHub struct represents an IoT Hub
+//
+// Resource information: https://azure.microsoft.com/en-us/services/iot-hub/
+// Pricing information: https://azure.microsoft.com/en-us/pricing/details/iot-hub/
+type IoTHub struct {
+	Address  string
+	Region   string
+	Sku      string
+	Capacity int64
+	DPS      bool
+
+	Operations *int64 `infracost_usage:"monthly_operations"`
+}
+
+func (r *IoTHub) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+var OperationsUsageSchema = []*schema.UsageItem{
+	{Key: "monthly_operations", DefaultValue: 0, ValueType: schema.Float64},
```

Comment:
I think it can be an integer value, as we can't have fractional amount of operations.
```suggestion
	{Key: "monthly_operations", DefaultValue: 0, ValueType: schema.Int64},
```